### PR TITLE
PD: Add the name field to the startup parameters 

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/18391
- Other reference link(s): ref https://github.com/tikv/pd/pull/8461

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: command-line-flags-for-scheduling-configuration.md [hunk 1]
```diff
@@ -57,6 +57,12 @@ The Scheduling node is used for providing the `scheduling` microservice for PD.
 - Default: `""`
 - If this flag is not set, logs are output to "stderr". If this flag is set, logs are output to the corresponding file.
 
+## `--name` <span class="version-mark">New in v8.3.0</span>
+
++ The name of the current Scheduling node.
++ Default: `"Scheduling-hostname"`
```
Comment (HuSharp) [inline command-line-flags-for-scheduling-configuration.md:63]:
```suggestion
+ Default: `"scheduling-${hostname}"`
```

File: command-line-flags-for-tso-configuration.md [hunk 2]
```diff
@@ -57,6 +57,12 @@ The TSO node is used for providing the `tso` microservice for PD. You can config
 - Default: `""`
 - If this flag is not set, logs are output to "stderr". If this flag is set, logs are output to the corresponding file.
 
+## `--name` <span class="version-mark">New in v8.3.0</span>
+
++ The name of the current TSO node.
++ Default: `"TSO-hostname"`
```
Comment (HuSharp) [inline command-line-flags-for-tso-configuration.md:63]:
```suggestion
+ Default: `"tso-${hostname}"`
```

**Review by HuSharp — APPROVED** [review]
rest LGTM~thx!

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**